control/autoware_mpc_lateral_controller/include/autoware_mpc_lateral_controller/lowpass_filter.hpp

```diff
@@ -12,15 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef MPC_LATERAL_CONTROLLER__LOWPASS_FILTER_HPP_
-#define MPC_LATERAL_CONTROLLER__LOWPASS_FILTER_HPP_
+#ifndef AUTOWARE_MPC_LATERAL_CONTROLLER__LOWPASS_FILTER_HPP_
+#define AUTOWARE_MPC_LATERAL_CONTROLLER__LOWPASS_FILTER_HPP_
 
 #include <algorithm>
 #include <cmath>
 #include <iostream>
 #include <vector>
 
-namespace autoware::motion::control::mpc_lateral_controller
+namespace autoware::motion::control::autoware_mpc_lateral_controller
```

Comment:
```suggestion
namespace autoware::motion::control::mpc_lateral_controller
```

In this case, you don't have to update namespace because `autoware::` has already existed. Please fix other part as well.

Reply:
fixed in 4cd8afcf85c3f3e12d615a120f7db84ec1d3ed50